packages/xrpl/test/runClientTests.ts

```diff
@@ -1,110 +0,0 @@
-/* eslint-disable mocha/no-setup-in-describe -- Necessary to programmatically generate tests */
```

Comment:
Did similar functionality move somewhere else? (Or does Jest handle this now?)

Reply:
@JST5000 was needed for mocha